functions/src/Services/Event.ts

```diff
@@ -1,12 +1,15 @@
-// const { App } = require('../app.js');
-// import * as App from 'firebase-admin';
 import { User } from "../Models/User";
 import { Event, eventConverter } from "../Models/Event";
 import { firestore } from 'firebase-admin';
 
-
+// Creates an event document and adds it to the events collection.
```

Comment:
I don't know that this needs a comment as I feel like everything here is Self explanatory. The only thing I would try to do different is see if it's possible to change the argument.
```suggestion
export const createEvent = (event: Event) => {
	firestore().collection('events').withConverter(eventConverter).add(event);
```

Reply:
Change the argument how?

Reply:
I gave an example of how it would look. It's in my comment as a suggested change.